mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-site_name: NGINX Ingress Controller
+site_name: NGINX-Ingress Controller
```

Comment:
This is the only non-md update in this PR.

I believe it should be the title of the [Project Homepage](https://kubernetes.github.io/ingress-nginx/), should be aligned as well.